Backend/PaymentService/BLL/Exceptions/AlreadyExistException.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace BLL.Exceptions
+{
+    public class AlreadyExistException : Exception
+    {
+        public AlreadyExistException() : base("Already Exist exception") { }
+        public AlreadyExistException(string message) : base(message) { }
```

Comment:
Structure like this for empty methods, ctors, classes:
```suggestion
        public AlreadyExistException() : base("Already Exist exception") 
        {
        }
        public AlreadyExistException(string message) : base(message) 
        {
        }
```